src/browser-driver-manager.js

```diff
@@ -1,259 +1,281 @@
+ * Read the installed version from the environment file.
+ * @returns {string|null} - The version if one exists. Null if errors are suppressed and no valid version exists.
+ * @throws {Error} - Environment file must exist.
+ * @throws {Error} - Environment file must have valid version.
  */
-async function verboseLog(chalk, verbose, msg) {
-  if (verbose) {
-    console.log(chalk.blue('log:'), msg);
+async function getVersion(suppressErrors = false) {
+  const pattern = /^VERSION="([\d.]+)"$/m;
+  const env = await getEnv();
+
+  if (!env) {
+    if (suppressErrors) {
+      return null;
+    }
+    throw new Error('No environment file exists. Please install first');
+  }
+
+  // Search for the pattern in the file path
+  const match = env.match(pattern);
+
+  if (!match || match.length < 2) {
+    if (suppressErrors) {
+      return null;
+    }
+    throw new Error(
+      `No version found in the environment file. Either remove the environment file and reinstall, or add a line 'VERSION={YOUR_INSTALLED_VERSION} to it.`
+    );
   }
+
+  const version = match[1];
+  return version;
 }
 
 /**
- * Uppercase first character of a string
- * @param {String} str - String to Titlecase
+ * Log the version to the console.
+ * @throws {Error} - Version must exist.
  */
-function titleCase(str) {
-  return str.charAt(0).toUpperCase() + str.substr(1);
+async function version() {
+  const version = await getVersion();
+  if (!version) {
+    throw new Error('No installed version found. Please install first');
+  }
+  console.log(version);
 }
 
 /**
- * Run the main bash script
- * @param {Object} options
- * @param {Boolean} [options.returnValue] - Return the bash script output rather than display it
- * @param {Boolean} [options.sudo] - Run the command as sudo
- * @param {String[]} options.args - List of script arguments
- * @returns {Promise<String|null>}
+ * Install a version of Chrome and Chromedriver.
+ * If already-installed Chrome and Chromedriver match this version, skip installation.
+ * If already-installed Chrome and Chromedriver are a different version, overwrite them.
+ * @param {string} browserId - Browser name (chrome), with an optional `@` and version number (e.g. 116.0.5845.96) or channel (e.g. beta, dev, canary)
+ * @param {object} options - Pass-through to installBrowser
+ * @throws {Error} Browser must be chrome
+ * @throws {Error} Browser platform must be detectable
+ * @throws {Error} Build ID must resolve
+ * @throws {Error} Browser must be uninstallable when one is already installed
+ * @throws {Error} Version of chrome and chromedriver must be findable
+ * @throws {Error} Environment file must be removable when one already exists
+ * @returns
  */
-function runBashScript({ returnValue = false, sudo = false, args } = {}) {
-  return new Promise((resolve, reject) => {
-    const lines = [];
-    let lastChunk;
+async function install(browserId, options) {
+  const [browser, version = 'latest'] = browserId.split('@');
```

Comment:
So it looks like `latest` in chrome for testing means `canary` which is not what we want. Looks like we'll need to use `stable` here to get latest released version.

```
» ./bin/browser-driver-manager install chrome                                    v18.17.1
Setting env CHROME/CHROMEDRIVER_TEST_PATH/VERSION
CHROME_TEST_PATH is set in /Users/stevenlambert/.browser-driver-manager/chrome/mac_arm-127.0.6522.0/chrome-mac-arm64/Google Chrome for Testing.app/Contents/MacOS/Google Chrome for Testing
CHROMEDRIVER_TEST_PATH is set in /Users/stevenlambert/.browser-driver-manager/chromedriver/mac_arm-127.0.6522.0/chromedriver-mac-arm64/chromedriver
VERSION: 127.0.6522.0
```

Version 127.0.6522.0 is [current canary](https://googlechromelabs.github.io/chrome-for-testing/) when it should install version 125.

```suggestion
  const [browser, version = 'stable'] = browserId.split('@');
```